system/security/shared/caching.cpp

```diff
@@ -580,7 +580,9 @@ bool CPermissionsCache::queryPermsManagedFileScope(ISecUser& sec_user, const cha
     if (m_managedFileScopesMap.empty())
     {
         *accessFlags = queryDefaultPermission(sec_user);
-        WARNLOG("Filescope managed scopes empty for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags,  msTick()-start);
+        if (m_secMgr) {
+            OWARNLOG("Filescope managed scopes empty for %s, applying default permissions %s(%d), took %dms", sec_user.getName(), getSecAccessFlagName(*accessFlags), *accessFlags, msTick() - start);
```

Comment:
Do you really want this coming out every time - I don't think the duplication provides any extra useful information?  I think it makes more sense for it to be in the code where the scopes are gathered.

Reply:
After more research, this case can happen if you aren't using managed scopes, which is allowed. The log message would become noise as you indicated. Removing the log stateme.